### DISCUSSION TITLE
Will wlroots-based stuff break when hyprland merges aquamarine support?

### DISCUSSION THREAD

**Discussion by mock1328** [hyprwm/Hyprland#6908] [Q&A] [answered]
I've heard, that the project is moving from wlroots to a custom library. I don't want to pollute the PR comments and notify all members, so decided to ask here: does this mean, that all wlroots-dependent stuff (e.g. screenshot tools, waybar, etc) will break and won't work with newer versions of `hyprland`? Sorry for my newbie question, I don't really understand how all these things binds together.

**Comment by fufexan** [comment, marked as answer]
No, they will still work, as the protocols will still be supported. It's only the rendering/input/etc backend that will change.

**Reply by mock1328** [reply]
Ack, thanks for explanation!

**Comment by vaxerski** [comment]
FWIW all the protocol implementations are already in hyprland since 0.41, so you've been unknowingly _not_ using wlroots for all these "wlroots-dependent" stuff already.
